pkg/controller/injector/k8s_binding_injector.go

```diff
@@ -59,35 +53,7 @@ func CreateK8sBindingInjector() (BindingInjector, error) {
 }
 
 func (b *k8sBindingInjector) Inject(binding *servicecatalog.Binding, cred *brokerapi.Credential) error {
-	is := makeInjectionSet(binding, cred)
-
-	if err := b.injectSecret(is.secret); err != nil {
-		return err
-	}
-
-	return nil
-}
-
-func (b *k8sBindingInjector) Uninject(binding *servicecatalog.Binding) error {
-	return fmt.Errorf("Not implemented")
-}
-
-func (b *k8sBindingInjector) injectSecret(s *v1.Secret) error {
-	cmc := b.client.Core().Secrets("default")
-	_, err := cmc.Create(s)
-	return err
-}
-
-func makeInjectionSet(binding *servicecatalog.Binding, cred *brokerapi.Credential) *injectionSet {
-	secret := makeSecret(binding, cred)
-
-	return &injectionSet{
-		secret: secret,
-	}
-}
-
-func makeSecret(binding *servicecatalog.Binding, cred *brokerapi.Credential) *v1.Secret {
-	return &v1.Secret{
+	secret := &v1.Secret{
 		ObjectMeta: v1.ObjectMeta{
 			Name: binding.Name,
```

Comment:
I don't think we'd pick up the name from SecretRef would we?  While I guess we could, I would think that if we're going to allow the user to pick the secret's name then it would be on the BindingSpec first, right?  Meaning, that's the source of truth and the name on/in the SecretRef is just a slave to it.  Right?